.github/workflows/main.yml

```diff
@@ -316,3 +316,157 @@ jobs:
         name: Realm.UnityWeaver.${{ steps.find-nupkg-version.outputs.package_version }}.nupkg
         path: ${{ github.workspace }}/Realm/packages/Realm.UnityWeaver.${{ steps.find-nupkg-version.outputs.package_version }}.nupkg
         retention-days: ${{ github.event_name != 'pull_request' && 30 || 1 }}
+  run-tests-net-framework:
+    runs-on: windows-latest
+    name: Run tests for NET framework
+    needs: build-packages
+    steps:
+    - name: Add msbuild to PATH
+      uses: microsoft/setup-msbuild@v1.0.2
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - name: Fetch Realm
+      uses: actions/download-artifact@v2
+      with:
+        name: Realm.${{ needs.build-packages.outputs.package_version }}.nupkg
+        path: ${{ github.workspace }}/Realm/packages/
+    - name: Fetch Realm.Fody
+      uses: actions/download-artifact@v2
+      with:
+        name: Realm.Fody.${{ needs.build-packages.outputs.package_version }}.nupkg
+        path: ${{ github.workspace }}/Realm/packages/
+    - name: Build the tests
+      run: msbuild ${{ github.workspace }}/Tests/Realm.Tests -p:RestoreConfigFile=${{ github.workspace }}/Tests/Test.NuGet.Config -p:Configuration=Release -p:UseRealmNupkgsWithVersion=${{ needs.build-packages.outputs.package_version }} -restore -p:TargetFramework=net461
+    - name: Run the tests
+      run: ${{ github.workspace }}/Tests/Realm.Tests/bin/Release/net461/Realm.Tests.exe --result=TestResults.Windows.xml --labels=After
+    - name: Publish Unit Test Results
+      uses: EnricoMi/publish-unit-test-result-action/composite@v1
```

Comment:
What happens if this repository disappears at some point?
Does it make sense to just fork it and point to our local copy within the Realm org?

Reply:
I wouldn't be too worried about it - we don't typically fork repos "just in case" - if it disappears we'll figure it out 😄

Reply:
👍